livekit-plugins/livekit-plugins-assemblyai/livekit/plugins/assemblyai/stt.py

```diff
@@ -66,9 +64,9 @@ def __init__(
         encoding: Optional[str] = "pcm_s16le",
         disable_partial_transcripts: bool = False,
         enable_extra_session_information: bool = False,
-        end_utterance_silence_threshold: int = 1000,
+        end_utterance_silence_threshold: Optional[int] = 100,
         http_session: Optional[aiohttp.ClientSession] = None,
-        buffer_size_seconds: float = 0.2,
+        buffer_size_seconds: float = 0.05,
```

Comment:
Not sure if this option is really relevant for the users. Let's pick the same one as DG?

Reply:
i didn't want to introduce additional latency, it seems that 50ms chunks is a good number for everything, wdyt?

Reply:
yeah 50ms seems good

Reply:
what I mean is that it is fine to hardcode it, but no big deal to leave it there